projects/packages/scheduled-updates/tests/php/WPCOM_REST_API_V2_Endpoint_Update_Schedules_Test.php

```diff
@@ -753,7 +753,10 @@ public function test_updating_autoupdate_plugins_on_delete() {
 
 		$this->assertTrue( $result->get_data() );
 		$this->assertSame( $expected_result, get_option( 'auto_update_plugins' ) );
-		$this->assertSame( 3, self::get_sync_counter() );
+		// Usually this should be 3, but in rare cases when the clock ticks over to the next second, the
+		// counter bumps 4 times. For now, we'll use this workaround to split the difference and accept
+		// both 3 and 4 as valid; outside of that something more catastrophic has occurred.
+		$this->assertContains( self::get_sync_counter(), array( 3, 4 ) );
```

Comment:
This works, although if it fails the error message is not all that descriptive
```
Failed asserting that an array contains 7.
```
we could do a little better with something like this
```suggestion
		$this->assertThat( self::get_sync_counter(), $this->logicalOr( $this->identicalTo( 3 ), $this->identicalTo( 4 ) ) );
```
which produces
```
Failed asserting that 7 is identical to 3 or is identical to 4.
```

Or we could just supply an appropriate `$message` argument.

Reply:
Works for me.